### PR TITLE
Modify describe to not list snapshots

### PR DESCRIPTION
**Pull request by DeepikaDixit**
## Change Overview

Listing snapshots is an `expensive` operation. Also, we do not need a list of snapshots.
Changing the command to check for latest instead

## Pull request type

Please check the type of change your PR introduces:
- [ ] :construction: Work in Progress
- [ ] :rainbow: Refactoring (no functional changes, no api changes)
- [ ] :hamster: Trivial/Minor
- [x] :bug: Bugfix
- [ ] :sunflower: Feature
- [ ] :world_map: Documentation
- [ ] :robot: Test

## Issues

- #XXX

## Test Plan

- [ ] :muscle: Manual
- [x] :zap: Unit test
- [ ] :green_heart: E2E

### CODE REVIEW

File: pkg/restic/restic.go [hunk 1]
```diff
@@ -255,9 +266,7 @@ func resticAzureArgs(profile *param.Profile, repository string) []string {
 
 // GetOrCreateRepository will check if the repository already exists and initialize one if not
 func GetOrCreateRepository(cli kubernetes.Interface, namespace, pod, container, artifactPrefix, encryptionKey string, profile *param.Profile) error {
-	stdout, stderr, err := listSnapshots(profile, artifactPrefix, encryptionKey, cli, namespace, pod, container)
-	format.Log(pod, container, stdout)
```
Comment (DeepikaDixit) [inline pkg/restic/restic.go]: We don't need to print the output. That will help reduce the long logs

Comment (DeepikaDixit) [reply pkg/restic/restic.go]: It is already printed in `getLatestSnapshots`

File: pkg/restic/restic.go [hunk 2]
```diff
@@ -99,6 +99,17 @@ func SnapshotsCommand(profile *param.Profile, repository, encryptionKey string)
 	return shCommand(command), nil
 }
 
+// LatestSnapshotCommand returns restic snapshots command for last snapshot
+func LatestSnapshotCommand(profile *param.Profile, repository, encryptionKey string) ([]string, error) {
```
Comment (pavannd1) [inline pkg/restic/restic.go]:
```suggestion
func LatestSnapshotsCommand(profile *param.Profile, repository, encryptionKey string) ([]string, error) {
```

**Comment by DeepikaDixit** [comment]
Needs +2 @pavannd1 @tdmanv

**Pull request merged**